portfolio/src/main/java/com/google/sps/servlets/LikeServlet.java

```diff
@@ -29,11 +29,12 @@
 @WebServlet("/comment/like")
 public class LikeServlet extends HttpServlet {
   private final DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+  private final String entityName = "Comment";
```

Comment:
Since this same entity name is shared across multiple classes, it should probably be pulled out to a common location that can be imported by both classes.

A common way to do this in Java is to define a `Constants.java` that contains only `static` constants.